alephclient/crawldir.py

```diff
@@ -79,15 +93,19 @@ def scandir(self, path: PathLike, id: str, parent_id: str):
 
     def get_foreign_id(self, path: PathLike) -> Optional[str]:
         if path == self.root:
-            if path.is_dir():
+            if path.is_dir():  # type: ignore
```

Comment:
Why not switch to requiring `Path` instead of `PathLike`? `PathLike` includes `str`, which would crash with this code. Plus it'll get rid of all the type ignore code smell in this.